landoapi/transplants.py

```diff
@@ -70,12 +70,13 @@ class LandingAssessmentState:
     """Encapsulates the state of a landing request for assessment.
 
     Holds fields that are necessary to assess if a landing is blocked,
-    but are not necessary to run lints on the entire stack.
+    but are not necessary to run checks on the entire stack. This includes
+    checks for permissions on a user to initiate a landing, whether the
+    requested path can land at the current time, etc.
     """
 
     auth0_user: A0User
-    revision_path: list[str]
-    revision_to_diff_id: dict[str, int]
+    landing_path_phid: list[tuple[str, int]]
```

Comment:
I don't think `landing_path_phid` is a good descriptor. This is a list of tuples, and the variable name implies a single value.

Reply:
This is the name that currently exists in Lando. It's just a landing path but instead of an `int` ID (ie `123` for `D123`) this variable uses the PHID. I'll update it to clarify.